atom/browser/api/atom_api_power_monitor.cc

```diff
@@ -55,6 +92,11 @@ v8::Local<v8::Value> PowerMonitor::Create(v8::Isolate* isolate) {
 void PowerMonitor::BuildPrototype(
     v8::Isolate* isolate, v8::Local<v8::FunctionTemplate> prototype) {
   prototype->SetClassName(mate::StringToV8(isolate, "PowerMonitor"));
+
+  mate::ObjectTemplateBuilder(isolate, prototype->PrototypeTemplate())
+      .MakeDestroyable()
+      .SetMethod("querySystemIdleState", &PowerMonitor::QuerySystemIdleState)
+      .SetMethod("querySystemIdleTime", &PowerMonitor::QuerySystemIdleTime);
```

Comment:
Why do we name these with "query" instead of using libcc's "calculate" nomenclature?

Reply:
Probably cause of my bad non-native-english-naming skill . 😅 open to suggestion if recommended to rename it.

Reply:
:smile: no worries!

My suggestion would be to copy the libchromium API names, e.g. `PowerMonitor::CalculateIdleState` and `powerMonitor.calculateIdleState`.

I don't feel very strongly about this; if other reviewers prefer `querySystem` I will go along

Reply:
OK, I'll wait for few other chime in and update PR if we'd like to go for chromium api names.